morp/interface/__init__.py

```diff
@@ -212,9 +255,24 @@ def _release(self, name, interface_msg, connection, **kwargs): raise NotImplemen
     def release(self, name, interface_msg, **kwargs):
         """release the message back into the queue, this is usually for when processing
         the message has failed and so a new attempt to process the message should be made"""
+        #interface_msg.raw.load()
         with self.connection(**kwargs) as connection:
-            self._release(name, interface_msg, connection=connection)
-            self.log("Message released back to {}", name)
+            delay_seconds = 0
+
+            cnt = interface_msg._count
+            if cnt:
+                delay_seconds = min(
+                    self.connection_config.options.get("max_timeout"),
+                    (cnt * 5) * cnt
```

Comment:
Magic Number! (5)

Is this just a backoff multiplier that seems to work well? Defining it somewhere as BACKOFF_MULTIPLIER would improve readability.

Should this be configurable?